pallets/capacity/src/lib.rs

```diff
@@ -100,6 +101,16 @@ pub mod pallet {
 	};
 	use sp_runtime::traits::{AtLeast32BitUnsigned, MaybeDisplay};
 
+	/// A reason for freezing funds.
+	#[pallet::composite_enum]
+	pub enum FreezeReason {
+		/// The account has staked tokens to the Frequency network.
+		#[codec(index = 0)]
+		Staked,
+		/// Funds are currently locked and are not yet liquid.
+		NotYetVested,
+	}
+
```

Comment:
Based on [this](https://paritytech.github.io/polkadot-sdk/master/frame_support/attr.pallet.html#composite-enum-palletcomposite_enum-optional), it looks like we do not need to duplicate all of the variants of this enum in each pallet it appears in; we only need the variants specific to the pallet, similar to `#[pallet::error]` and `#[pallet::event]`

Reply:
Thanks @JoeCap08055, based on those docs, I've refactored FreezeReason in both pallets.